docs/overview.rst

```diff
@@ -161,58 +164,54 @@ FAQ
 
-Before starting to create yet another data storage service, we had a long
-look to the existing solutions, to see if that would make sense to extend
-the community effort rather than re-inventing the wheel.
-
-It appeared that solutions we looked at weren't solving the problems we had,
-especially regarding fine-grained permissions.
-
-To see how Kinto compares to these solutions,
-read :ref:`the comparison table <comparison>`.
+To see how Kinto compares to CouchDB & Remote Storage, read :ref:`the comparison table <comparison>`.
 
 Can I encrypt my data?
 ----------------------
 
-Kinto server stores any data you pass to it, be it encrypted or not.
-We make it easy to use encryption in our Kinto.js client
-`using transformers <http://kintojs.readthedocs.org/en/latest/api/#transformers>`_.
+Kinto server stores any data you pass to it, whether it's encrypted or not. We believe
+encryption should always be done on the client-side, and we make it easy to use encryption in
+our Kinto.js client `using transformers <http://kintojs.readthedocs.org/en/latest/api/#transformers>`_.
 
 Is there a package for my Operating System?
 -------------------------------------------
 
-No, but it's a great idea. Packaging is hard and we're a small team, so if
-you'd like to help us out by maintaining packages for your favourite OS,
-we'd be delighted to collaborate with you!
+No, but it's a great idea. Maintaining packages for several platforms is time-consuming
+and we're a small team. At this time we're just making sure it's easy to run our server
+using our Makefile or our Dockerfile.
 
-That said, Kinto is :ref:`easy to install with pip <installation>` and
+Kinto is :ref:`easy to install with pip <installation>` and
 we've got `an image set up <https://hub.docker.com/r/kinto/kinto-server/>`_
 on the Docker hub, too.
 
+But if you'd like to help us out by maintaining packages for your favourite OS,
+we'd be delighted to collaborate with you!
+
+
 Why did you chose to use Python rather than X?
 ----------------------------------------------
 
-We know and love `Python <python.org>`_ for its simplicity and short
-learning curve, so it was an obvious choice for the development team. In
-addition, the Operations team at Mozilla is comfortable with deploying and
+We love `Python <python.org>`_ because it's a concise & expressive
+language with powerful data structures & easy to learn,
+so it was an obvious choice for the development team.
+
+In addition, the Operations team at Mozilla is comfortable with deploying and
 managing Python applications in production.
 
-However, the protocol and concepts behind Kinto don't rely on Python *per
-se*, so it is possible to have other Kinto implementations using other
-languages.
+However, Python is just an implementation detail *per se*. Kinto is
+defined by an HTTP protocol that could be implemented in any language.
+
 
 Is it Web Scale?
 ----------------
 
 YES™. Have a look at the ``/dev/null`` backend. ;-)
```

Comment:
Now that I see this... This is a fun answer, but I wonder if it serves our cause here :)